services/supervisord/supervisord.go

```diff
@@ -711,13 +711,13 @@ command =
         cfg:default.server.domain="{{ .PMMServerAddress }}"
         cfg:default.auth.generic_oauth.enabled=true
         cfg:default.auth.generic_oauth.name="Percona Account"
-        cfg:default.auth.generic_oauth.client_id="{{ .PerconaSSODetails.ClientID }}"
-        cfg:default.auth.generic_oauth.client_secret="{{ .PerconaSSODetails.ClientSecret }}"
+        cfg:default.auth.generic_oauth.client_id="{{ .PerconaSSODetails.GrafanaClientID }}"
```

Comment:
Do we need client_secret?

Reply:
the grafana config in the new spec doesn't mention it  

https://confluence.percona.com/pages/viewpage.action?spaceKey=PMM&title=Single+Sign-On+-+PMM+Server+several+domains+support#SingleSignOnPMMServerseveraldomainssupport-Orgdchanges

and this ticket went through QA and was working fine so I guess it should be ok